sdk/src/Services/S3/Generated/Model/Internal/MarshallTransformations/PutObjectRetentionRequestMarshaller.cs

```diff
@@ -97,7 +97,7 @@ public IRequest Marshall(PutObjectRetentionRequest publicRequest)
             {   
                 if (publicRequest.IsSetRetention())
                 {
-                    xmlWriter.WriteStartElement("ObjectLockRetention", "http://s3.amazonaws.com/doc/2006-03-01/");
+                    xmlWriter.WriteStartElement("Retention", "http://s3.amazonaws.com/doc/2006-03-01/");
```

Comment:
I assume this was a benign bug that we were using the wrong root element name. I see in the docs `Retention` is the correct value.